src/screens/ShipItemDetails/index.tsx

```diff
@@ -66,8 +64,17 @@ const ShipItemDetails = () => {
   };
 
   const submitShipmentDetail = (id: string) => {
+    if (Number(state.quantityPicked) > Number(item.quantityRemaining)) {
+      showPopup({
+        message: 'Quantity Picked is higher than quantity remaining',
+        positiveButton: {
+          text: 'Ok',
+        },
+      });
+      return;
+    }
     const request = {
-      'container.id': state.container.id,
+      'container.id': state?.container?.id ?? '',
```

Comment:
It cannot be `null`, hence no need to fallback to empty string.

Reply:
okay